sites/funmooc/src/backend/templates/richie/base.html

```diff
@@ -20,3 +20,9 @@
 {% endblock topbar_contact %}
 
 {% block body_footer_title %}{% endblock body_footer_title %}
+{% block body_js %}
+    <script type="text/javascript">window.__funmooc_marketing_context__ = JSON.parse('{{ MARKETING_CONTEXT|safe }}');</script>
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/gh/AmauriC/tarteaucitron.js@20210306/tarteaucitron.js"></script>
```

Comment:
I would prefer to vendor it :man_shrugging:

Reply:
If we want to vendor it, we have several files to include (10+) in the project which are not minified...

Reply:
So finally, I installed tarteaucitronjs from yarn in `frontend` directory. As `tarteaucitron` code base is ready to use, I copy the `tarteaucitronjs` from `node_modules` to your static folder.
In this way, we do not have to embed source of this package in the repository and we are able to upgrade version easily.

Reply:
<img src="https://media.giphy.com/media/WOs3HWNVBIS1VLlaxj/giphy.gif">